rbc/npy_mathimpl.py

```diff
@@ -70,6 +70,26 @@ def impl(x, y):
     return context.compile_internal(builder, impl, sig, args)
 
 
+def np_signbit_impl(context, builder, sig, args):
+
+    def impl(x):
+        if x < 0:
+            return True
+        return False
```

Comment:
Float zero has also signbit, e.g. -0.0 and 0.0 are different float values. See how signbit is implemented in numpy: https://github.com/numpy/numpy/blob/master/numpy/core/src/npymath/_signbit.c . 
Is it possible to implement the union trick here?

Reply:
I copied this code on godbold.com and implemented the IR it outputs. See: https://godbolt.org/z/P5Wj4b